.eslintrc.js

```diff
@@ -1,3 +1,5 @@
+const { jest } = require('@jest/globals');
```

Comment:
A test is complaining about this file:

```
 ESLint: 8.57.0

Error: Cannot read config file: /home/runner/work/renovate/renovate/.eslintrc.js
Error: Do not import `@jest/globals` outside of the Jest test environment
    at Object.<anonymous> (/home/runner/work/renovate/renovate/node_modules/.pnpm/@jest+globals@29.7.0/node_modules/@jest/globals/build/index.js:12:7)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/home/runner/work/renovate/renovate/.eslintrc.js:1:18)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
 ELIFECYCLE  Command failed with exit code 2.
Error: Process completed with exit code 2.
```

Reply:
ups, vscode auto import, will revert on next push